src/components/Careers/Careers.tsx

```diff
@@ -63,7 +66,7 @@ const Careers = () => {
           register={register}
           onChange={() => checkboxHandler(name as CareerCheckboxesCategory)}
         >
-          {value}
+          {t(value)}
```

Comment:
Don't translate when sending props, translate the props themself in the component!